src/features/expansion/tilde_expansion/tilde_expansion.c

```diff
@@ -100,6 +84,22 @@ int		expand_assignments(char **assignment)
 	return (EXIT_SUCCESS);
 }
 
+int		expand_internal_variables(t_process *p)
+{
+	t_assignment	*assign_tmp;
+	int				status;
+
+	assign_tmp = p->first_assignment;
+	status = 0;
+	while (assign_tmp != NULL)
+	{
+		if (ft_strchr(assign_tmp->data, '=') != NULL)
```

Comment:
so if there is '=' ,then there is no tilde expansion yeh? :)

Reply:
No, the expansion will happen if there is a =.
Might be an unneccessary check though

Reply:
ty